src/Phinx/Db/Adapter/SQLiteAdapter.php

```diff
@@ -798,7 +814,7 @@ protected function calculateNewTableColumns($tableName, $columnName, $newColumnN
         $selectColumns = array_map([$this, 'quoteColumnName'], $selectColumns);
         $writeColumns = array_map([$this, 'quoteColumnName'], $writeColumns);
 
-        if (!$found) {
+        if (!$found && !empty($columnName)) {
```

Comment:
!empty() is not needed here.

Reply:
Changed to `!==` as I think that's what you mean by "non-cloaking"?

Reply:
No, i was talking about the equivalent of !empty() but without cloaking. Fixed it for you.